src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.html

```diff
@@ -35,7 +35,7 @@
     <ng-template ngbNavContent>
       <cd-grafana [grafanaPath]="'host-overview?'"
                   uid="y0KGL0iZz"
-                  grafanaStyle="two">
+                  grafanaStyle="three">
```

Comment:
old size is working:
```suggestion
                  grafanaStyle="two">
```

Reply:
@tspmelo how much bigger should four be?

Reply:
`1160px` works for both panels that will use it.

Reply:
@tspmelo which style is preferable for the host-details component between "two" and "four"?

Reply:
`hosts.component.html` is `two`,
`host-details.component.html` is `four`.

Reply:
Thank you